lib/gui/components/safe-webview.js

```diff
@@ -46,6 +46,22 @@ const VERSION_PARAM = 'etcher-version';
 const ELECTRON_SESSION = 'persist:success-banner';
 
 /**
+ * @summary API version search parameter key
+ * @constant
+ * @private
+ * @type {String}
+ */
+const API_VERSION_PARAM = 'api-version';
```

Comment:
I guess you could also explain that the HTML banner can use this version-number to determine which features the safe-webview supports (independent of the Etcher version-number).

(perhaps it's just me, but 'API' feels like the wrong term for this)